static/docs/get-started/add-files.md

```diff
@@ -41,7 +41,9 @@ $ git commit -m "add source data to DVC"
 ### Expand to learn about DVC internals
 
 You can see that actual data file has been moved to the `.dvc/cache` directory
-(usually hardlink or reflink is created, so no physical copying is happening).
+(ideally with reflinks if available on the system, otherwise by file copy – see
```

Comment:
let's make it a separate `<details>` section - something like `Important performance caveats` or something like this.

Reply:
Ok, done. This doc may have too many expandable detail sections now though, you may want to see it in the review app: https://dvc-org-pr-345.herokuapp.com/doc/get-started/add-files